ui/src/components/inputs/EditorView.vue

```diff
@@ -507,6 +506,7 @@
     import MetadataEditor from "../flows/MetadataEditor.vue";
     import {useFlowOutdatedErrors} from "./flowOutdatedErrors";
     import {usePluginsStore} from "../../stores/plugins";
+    import {localizeElementAtIndex} from "@kestra-io/ui-libs/flow-yaml-utils";
```

Comment:
For consistency, should we use the other import pattern?  
Use `import * as YAML_UTILS from "..."`